ext/bindings/message.hpp

```diff
@@ -21,6 +21,7 @@ namespace pulsar_rb {
   class Message {
   public:
     pulsar::Message _msg;
+    bool built = false; // built with builder rather than returned from c++
```

Comment:
Given the comment you have written for getTopicName above, shall we name this `received` instead? What do you think?

Reply:
name it `received`, default it  to `true`, and set it to `false` when we do the building?
sure.

Reply:
Yes, that should do it.